app/controllers/categories_controller.rb

```diff
@@ -286,9 +286,13 @@ def category_params
       end
 
       if SiteSetting.tagging_enabled
-        params[:allowed_tags] ||= []
-        params[:allowed_tag_groups] ||= []
-        params[:required_tag_group_name] ||= ''
+        params[:allowed_tags] = params[:allowed_tags].presence || [] if params[:allowed_tags]
+        params[:allowed_tag_groups] = params[:allowed_tag_groups].presence || [] if params[:allowed_tag_groups]
+        params[:required_tag_group_name] = params[:required_tag_group_name].presence || '' if params[:required_tag_group_name]
+      end
+
+      if SiteSetting.enable_category_group_moderation?
+        params[:reviewable_by_group_id] = Group.find_by(name: params[:reviewable_by_group_name])&.id if params[:reviewable_by_group_id]
```

Comment:
We can just pluck the id here. Also, I think we should provide better feedback if `reviewable_by_group_id` is an invalid group id.

Reply:
Sorry just realized this is not related to your change but since we're making changes here we might as well fix it grin